gr-uhd/examples/grc/uhd_normalized_gain.grc

```diff
@@ -1,6 +1,7 @@
 options:
   parameters:
     author: ''
+    catch_exceptions: 'True'
```

Comment:
for our edification, why are there so many changes in this GRC script?

Reply:
He regenerated the flow graph to just change gain type back to normalized. But it's an old flow graph, so you get all the changes made to GNU Radio in addition.

Reply:
that's my guess too ... OK so if that's the case I'd prefer these to be 2 separate commits ... since these are not required for the others -- the main purpose of this PR -- to work ... assuming my note here correct that is ... if they are required to be together, please note as such and I'll be OK with that. THx!